Formula/lib/liblc3.rb

```diff
@@ -0,0 +1,81 @@
+class Liblc3 < Formula
+  desc "Low Complexity Communication Codec library and tools"
+  homepage "https://github.com/google/liblc3"
+  url "https://github.com/google/liblc3/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "958725e277685f9506d30ea341c38a03b245c3b33852cd074da6c8857525e808"
+  license "Apache-2.0"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  uses_from_macos "python"
+
+  def install
+    # disable tools build due to rpath issue, see https://github.com/google/liblc3/pull/53
```

Comment:
Since they seemed hesitant to merge it, it might be better to create an issue with the problem to link to?